### PR TITLE
CAL-265 Fixed overflow issues in Video Admin Plugin

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR fixes overflow issues in the Video Admin Plugin
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@glenhein 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@andrewkfiedler 
@jlcsmith
#### How should this be tested?
Build / Install Experimental / Configure Stream with very large name and observe overflow handling
#### What are the relevant tickets?
[CAL-265](https://codice.atlassian.net/browse/CAL-265)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/video/video-admin-plugin/src/main/webapp/css/index.css [hunk 1]
```diff
@@ -10,11 +10,27 @@
...
 }
 
+table {
+    table-layout: fixed;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+td {
+    overflow: hidden;
```
Comment (andrewkfiedler) [inline catalog/video/video-admin-plugin/src/main/webapp/css/index.css]: I would add a max-width on these `td` elements: 
```
    max-width: 200px;  // or some other value
```

File: catalog/video/video-admin-plugin/src/main/webapp/css/index.css [hunk 2]
```diff
@@ -10,11 +10,27 @@
...
+    width: 20%;
+}
+
 .td-btn {
     border: 1px solid transparent;
     border-radius: 4px;
 }
 
+table {
+    table-layout: fixed;
```
Comment (andrewkfiedler) [inline catalog/video/video-admin-plugin/src/main/webapp/css/index.css]: I think it'd be better to leave out this `table` block of styling as well as the `.td-name` styling.

Instead, I would add a style that applies to the `main` element.

```
main {
   overflow: auto;
}
```

This combined with the `max-width` on the `td` elements should result in a view that will accommodate extremes fairly well.

Comment (bdeining) [inline catalog/video/video-admin-plugin/src/main/webapp/css/index.css]: Done

File: catalog/video/video-admin-plugin/src/main/webapp/css/index.css (hunk 1 above)
Comment (bdeining) [inline catalog/video/video-admin-plugin/src/main/webapp/css/index.css]: Done

**Comment by jlcsmith** [comment]
built/tested - LGTM

**Pull request merged**